corehq/apps/userreports/mixins.py

```diff
@@ -117,3 +120,14 @@ def get_db_column_ids(self, column_id):
         else:
             # if the column isn't found just treat it as a normal field
             return [column_id]
+
+
+class NoPropertyTypeCoercionMixIn(object):
```

Comment:
Would it make sense to have something like this in jsonobject code?

Reply:
good question. @dannyroberts any opinions?

Reply:
Yeah, I don't have a super strong opinion about it, just that I never had a strong feel for what would and wouldn't be worth adding as other base classes/mixins. For example there's `_allow_dynamic_properties`, and there's a sort of "strict" version of the default string conversions that deals with datetimes better, which lives currently in dimagi.ext.jsonobject (and which everywhere in our code, including here, imports from).

If someone wanted to add those to the jsonobject library's api as mixins called `jsonobject.NoStringConversions`, `jsonobject.StrictStringConversions`, and `jsonobject.DisallowDynamicProperties` or some such thing, I think I'd be into it.